OpenSim/Actuators/Millard2012EquilibriumMuscle.cpp

```diff
@@ -55,6 +55,8 @@ void Millard2012EquilibriumMuscle::constructProperties()
     constructProperty_ForceVelocityCurve(ForceVelocityCurve());
     constructProperty_FiberForceLengthCurve(FiberForceLengthCurve());
     constructProperty_TendonForceLengthCurve(TendonForceLengthCurve());
+
+    setMinControl(get_minimum_activation());
```

Comment:
This only sets the default value for `min_control`. Deserialization populates the Object properties and overwrites the property values for those specified in the model (XML) file.